Bottom: 5c2d2f23ac026c0e64fc87996522232d7800ccb2
Top:    5350a91559bb45c185d8e8c9387ba6969135df44
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-01 01:36:35 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index fc39ec7..93653b6 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -306,7 +306,7 @@ define KernelPackage/sound-soc-rt5350-le89156
   FILES:=\
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-le89156.ko \
 	$(LINUX_DIR)/sound/soc/ralink/snd-soc-rt5350-le89156.ko
-  AUTOLOAD:=$(call AutoLoad,57,snd-soc-rt5350-le89156)
+  AUTOLOAD:=$(call AutoLoad,57,snd-soc-le89156 snd-soc-rt5350-le89156)
   DEPENDS:=@TARGET_ramips +kmod-sound-soc-rt5350
   $(call AddDepends/sound)
 endef
